src/api/java/net/earthcomputer/multiconnect/api/EnumProtocol.java

```diff
@@ -10,6 +10,7 @@
 
     // IMPORTANT: only add to the end of this enum - do not insert in the middle for new versions
     AUTO("Auto", -1),
+    V1_15_2("1.15.2", Protocols.V1_15_2),
```

Comment:
Please only add constants to the bottom of this enum

Reply:
Resolved (RIP OCD Users looking at the code xD)